### PR TITLE
Add quiz screen

### PR DESCRIPTION
**Pull request by narendrasss**

### CODE REVIEW

**Comment by anhduc130** [comment]
The gravity form should be embedded in the quiz screen.

**Comment by narendrasss** [comment]
Thanks @anhduc130. That means this PR is now ready for QA.

File: templates/vue/src/components/lightbox/quiz/Question.vue [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+            .querySelector(`form#gform_${id}`)
+            .addEventListener("submit", (event) => {
+              this.formOpened = false
+              this.$emit("form-submitted")
+              this.disableButton = false;
+            })
+
+          this.$refs["form-container"].appendChild(gravityForm)
+          this.formOpened = true
+          this.$emit("form-opened")
```
Comment (narendrasss) [inline templates/vue/src/components/lightbox/quiz/Question.vue]: We should avoid doing any direct manipulation of the DOM. Vue has a `v-html` attribute that you can use to inject html, and events "bubble" so you can attach the submit handler to the `form-container` element and it should work fine.

Comment (anhduc130) [reply templates/vue/src/components/lightbox/quiz/Question.vue]: Please feel free to modify the code. Thanks.

File: templates/vue/src/App.vue [hunk 2]
```diff
@@ -22,37 +22,45 @@ export default {
 
 html {
   font-size: 100%;
+}
```
Comment (wynnset) [inline templates/vue/src/App.vue]: It seems like this is reverting the SASS changes. Could you please make sure this is written is SCSS instead and not revert that change?

File: templates/vue/src/components/lightbox/quiz/Question.vue (hunk 1 above)
Comment (wynnset) [reply templates/vue/src/components/lightbox/quiz/Question.vue]: Yes, I agree with Narendra here. We should avoid doing things through plain JS when there is a vue way of doing it. Please fix this.

File: templates/vue/src/App.vue (hunk 2 above)
Comment (narendrasss) [reply templates/vue/src/App.vue]: Sorry about that, looks like a problem from doing the merge. Fixed.

**Pull request merged**